CHANGELOG-helmop.md

```diff
@@ -1,3 +1,11 @@
+## 0.7.2
```

Comment:
We write our changelog (and determine the next version) just before we make the release so adding this is not necessary.